apps/src/feedbackBlocks.js

```diff
@@ -85,6 +86,14 @@ FeedbackBlocks.prototype.show = function() {
   }
 };
 
+FeedbackBlocks.prototype.hideDiv = function() {
+  this.div.className += " hidden";
+};
+
+FeedbackBlocks.prototype.revealDiv = function() {
+  this.div.className = this.div.className.replace(/\hidden\b/,'');
```

Comment:
hah, oh gosh, that's a typo. Supposed to be `/\bhidden\b/`; `\b` enforces a word boundary. so it wouldn't select "hiddenSomethingElse".

I was debating exactly that; it's probably safe to use jQuery in this context, but I wanted to avoid adding a dependency to this class if I didn't have to.

This regex is pretty functionally similar to YMNNJq's version, and is I believe a common approach. Would a comment just explaining the regex be sufficient?